c++/src/H5Exception.h

```diff
@@ -75,7 +75,7 @@ class H5_DLLCPP Exception {
     Exception(const Exception &orig);
 
     // virtual Destructor
-    virtual ~Exception() throw();
+    virtual ~Exception() noexcept = default;
```

Comment:
As of C++11, destructors are implicitly `noexcept` _unless the generated trivial destructor would not be_. IOW, unless a base or member has a destructor that is `noexcept(false)`.

Reply:
OK, so you're saying I should just remove the `noexcept` keyword here?

Reply:
Upon further research and investigation, I agree, and have made this change.